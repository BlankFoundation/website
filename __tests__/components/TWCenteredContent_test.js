import React from 'react';
import TestRenderer from 'react-test-renderer';
import TWCenteredContent from '../../components/TWCenteredContent';

describe("TWCenteredContent", () => {
  it('renders correctly', () => {
    const rendered = TestRenderer.create(<TWCenteredContent />).toJSON();
    expect(rendered).toMatchSnapshot()
  });
});