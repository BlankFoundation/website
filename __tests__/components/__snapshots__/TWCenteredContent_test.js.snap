// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TWCenteredContent renders correctly 1`] = `
<div
  className="flex items-center justify-center flex-grow"
>
  <div
    className="inline-block"
  >
    <div>
      child1
    </div>
    <div>
      child2
    </div>
    <div>
      child3
    </div>
  </div>
</div>
`;
